[itk] download failure

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
./vcpkg.exe install itk
```


### Failure logs

Package: itk:x64-windows -> 5.3-rc02#6

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2023-09-15-ac02a9f660977426b8ec6392919fbb1d51b10998
    vcpkg-scripts version: 486a4640d 2023-10-13 (3 days ago)

**To Reproduce**

`vcpkg install itk`
**Failure logs**

```
-- Downloading https://github.com/InsightSoftwareConsortium/ITK/archive/v5.3-rc02.tar.gz -> InsightSoftwareConsortium-ITK-v5.3-rc02.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Disabling metrics because vcpkg.disable-metrics exists
[DEBUG] Trying to load bundleconfig from C:\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: C:\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
Downloading https://github.com/InsightSoftwareConsortium/ITK/archive/v5.3-rc02.tar.gz
error: Failed to download from mirror set
error: https://github.com/InsightSoftwareConsortium/ITK/archive/v5.3-rc02.tar.gz: failed: status code 404
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1051): 
[DEBUG] Time in subprocesses: 0us
[DEBUG] Time in parsing JSON: 9us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 119us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 607 ms (606379us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
      
      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:
      
      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.
      
      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.
      
      3. Your proxy's remote server is out of service.
      
      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.
      

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  ports/itk/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:168 (include)



```


### Additional context

It's possible, that error is in vcpkg script, which tries to get release from

https://github.com/InsightSoftwareConsortium/ITK/archive/v5.3-rc02.tar.gz

instead of

https://github.com/InsightSoftwareConsortium/ITK/releases/download/v5.3rc02/InsightData-5.3rc02.tar.gz

(Link is taken from release page)

--- a/ports/itk/portfile.cmake
+++ b/ports/itk/portfile.cmake
@@ -3,8 +3,8 @@ vcpkg_buildpath_length_warning(37)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO InsightSoftwareConsortium/ITK
-    REF v5.3-rc02
-    SHA512 fe703bc6ed681cb9983d7d6e21c8ffa7650337e470c09a7241de58a463c23e315516b1a81a18c14f682706056a0ec66932b63d2e24945bdcea03169bc1122bb2
+    REF "v${VERSION}"
+    SHA512 48e38864d7cd20b4ff23cfca1a29b4cbf453ce842e037ca473ce5c7e9a5b1c0bf6b12e4c544c812bff2b4bb9feca409587175b6570929c9ac172cb4402d679da
     HEAD_REF master
     PATCHES
         double-conversion.patch

--- a/ports/itk/vcpkg.json
+++ b/ports/itk/vcpkg.json
@@ -1,7 +1,6 @@
 {
   "name": "itk",
   "version-string": "5.3rc02",
-  "port-version": 7,
   "description": "Insight Segmentation and Registration Toolkit (ITK) is used for image processing and analysis.",
   "homepage": "https://github.com/InsightSoftwareConsortium/ITK",
   "license": "Apache-2.0",

--- a/ports/itk/vcpkg.json
+++ b/ports/itk/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "itk",
-  "version": "5.3-rc02",
-  "port-version": 6,
+  "version-string": "5.3rc02",
+  "port-version": 7,
   "description": "Insight Segmentation and Registration Toolkit (ITK) is used for image processing and analysis.",
   "homepage": "https://github.com/InsightSoftwareConsortium/ITK",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3526,7 +3526,7 @@
     },
     "itk": {
       "baseline": "5.3rc02",
-      "port-version": 7
+      "port-version": 0
     },
     "itpp": {
       "baseline": "4.3.1",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3525,8 +3525,8 @@
       "port-version": 0
     },
     "itk": {
-      "baseline": "5.3-rc02",
-      "port-version": 6
+      "baseline": "5.3rc02",
+      "port-version": 7
     },
     "itpp": {
       "baseline": "4.3.1",

--- a/versions/i-/itk.json
+++ b/versions/i-/itk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b74bb68b58026e759112afb8155d44650cee24b4",
+      "version-string": "5.3rc02",
+      "port-version": 0
+    },
+    {
       "git-tree": "9da4bc8949edd65bc3e0afa8e694480e330f5f40",
       "version": "5.3-rc02",
       "port-version": 6

--- a/versions/i-/itk.json
+++ b/versions/i-/itk.json
@@ -1,11 +1,6 @@
 {
   "versions": [
     {
-      "git-tree": "6a73ecdf61dd3e1e20fb392912bf4b15e4bab9dc",
-      "version-string": "5.3rc02",
-      "port-version": 7
-    },
-    {
       "git-tree": "9da4bc8949edd65bc3e0afa8e694480e330f5f40",
       "version": "5.3-rc02",
       "port-version": 6

--- a/versions/i-/itk.json
+++ b/versions/i-/itk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6a73ecdf61dd3e1e20fb392912bf4b15e4bab9dc",
+      "version-string": "5.3rc02",
+      "port-version": 7
+    },
+    {
       "git-tree": "9da4bc8949edd65bc3e0afa8e694480e330f5f40",
       "version": "5.3-rc02",
       "port-version": 6

